recipe/meta.yaml

```diff
@@ -38,6 +37,8 @@ requirements:
     - zlib
 
 test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
Is this line still needed?

Reply:
This what gets installed in `vc9` test environment (https://ci.appveyor.com/project/conda-forge/libgit2-feedstock/build/1.0.81/job/pytyi6qnjtk0t6ex#L1889) when this is present, mixture of `vc9` and `vc14`.

```
## Package Plan ##
  environment location: C:\bld\libgit2_1528674472804\_test_env
The following NEW packages will be INSTALLED:
    curl:           7.60.0-vc9_0      conda-forge [vc9 ]
    krb5:           1.14.6-vc9_0      conda-forge [vc9 ]
    libgit2:        0.27.1-h3e54e04_0 local      
    libssh2:        1.8.0-vc14_1      conda-forge [vc14]
    python:         3.6.5-1           conda-forge
    vc:             9-0               conda-forge
    vs2008_runtime: 9.0.30729.6161-0  conda-forge
    vs2015_runtime: 14.0.25420-0      conda-forge
    zlib:           1.2.11-vc14_0     conda-forge [vc14]
```